web/stripe-template/api/routes/webhook/POST-subscription.ts

```diff
@@ -2,26 +2,28 @@ import { RouteContext } from "gadget-server";
 import { getStripeWebhookEvent } from "../../stripe";
 import { objKeyConvert, destructure } from "../../utils";
 
-/**
- * Route handler for POST webhook/subscription
- *
- * @param { RouteContext } route context - see: https://docs.gadget.dev/guides/http-routes/route-configuration#route-context
- *
- */
+export type StripeWebhookEvent = {
+  type: string;
+  data: {
+    object: any;
+  };
+};
+
 export default async function route({
```

Comment:
I think we should do `const route = async (...` and `export default route;` to match what we give you when a new route is created in Gadget, but this does work so up to you

Reply:
Changed